### ISSUE TITLE
Latest lines not shown on right layout

### ISSUE THREAD

**Issue by underdpt** [noseglid/atom-build#468]
Hi,
When using right layout, I'm missing the latest 4 lines of the output. In this image, it should output 5 lines from 1 to 5, but only 1 is visible.
![example](https://cloud.githubusercontent.com/assets/8122137/18203408/442b3928-7117-11e6-8cef-5597b0050f0e.jpg)

Also note that if I change the layout to top or bottom, and then moves again to right layout, it seems to work correctly.

I'm also missing the right bar and have to use the mouse wheel to move around the output.

Thanks for this great work!

**Comment by noseglid** [comment]
Definitely a bug with the CSS. The `.build .output` panel seem to only grow (not shrink). For example; starting with a window size of 600 px, and the output is fine. Increasing the height to 800 px and it's still good (but you have to re-toggle the build window to calculate new heights). Shrinking back to 600 px, and the CSS height remains at 800 px.

I'm not a good CSS ninja. Did some fiddling but didn't get it to work. If anyone with some CSS skills wants to give this a shot, please go ahead.

**Comment by underdpt** [comment]
In the meantime, what about adding some blank lines at the end of the output for vertical layout? It's not pretty but can make useable the plugin on that configuration.

**Comment by underdpt** [comment]
Or allowing to scroll past end of buffer...

**Comment by noseglid** [comment]
Not fond of adding blank lines.
Scrolling past end may work.
Otherwise, as a workaround, maybe you could use top/bottom orientation until this is fixed?

**Comment by underdpt** [comment]
Yes, that's what i'm doing now, but I am wasting a lot of vertical space with 16:9 screens :-)
